packages/icons-vue/tasks/build.js

```diff
@@ -5,13 +5,102 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-/* eslint-disable no-console */
-
 'use strict';
 
-const path = require('path');
-const build = require('../src/build');
+const meta = require('@carbon/icons/build-info.json');
+const { rollup } = require('rollup');
+const babel = require('rollup-plugin-babel');
+const virtual = require('./plugins/virtual');
+const { createIconComponent } = require('./createIconComponent');
+
+const BUNDLE_FORMATS = [
+  {
+    format: 'esm',
+    dir: 'es',
+  },
+  {
+    format: 'cjs',
+    dir: 'lib',
+  },
+];
+
+const external = ['@carbon/icon-helpers'];
+const babelConfig = {
+  babelrc: false,
+  exclude: /node_modules/,
+  presets: [
+    [
+      '@babel/preset-env',
+      {
+        targets: {
+          browsers: ['extends browserslist-config-carbon'],
+        },
+      },
+    ],
+  ],
+};
+
+/**
+ * Take the processed `meta.json` file from `@carbon/icons` and generate an
+ * entrypoint for `@carbon/icons-vue`. This entrypoint is generated by
+ * building icon components built on top of the `<Icon>` primitive in `src`.
+ */
+async function build() {
+  const modules = meta.map(icon => {
+    const { source } = createIconComponent(icon.moduleName, icon.descriptor);
+    return {
+      moduleName: icon.moduleName,
+      filepath: icon.outputOptions.file.replace(/^es\//g, '').slice(0, -3),
+      source,
+    };
+  });
+
+  const entrypoint = `export const CarbonIconsVue = {
+  install(Vue, options) {
+    const { components } = options;
+    Object.keys(components).forEach(key => {
+      Vue.component(key, components[key]);
+    });
+  },
+}`;
+
+  const bundle = await rollup({
+    input: {
+      index: '__entrypoint__.js',
+      ...modules.reduce(
+        (acc, { filepath, moduleName }) => ({
+          ...acc,
+          [filepath]: `${moduleName}.js`,
+        }),
+        {}
+      ),
+    },
+    external,
+    plugins: [
+      virtual({
+        '__entrypoint__.js': entrypoint,
+        ...modules.reduce(
+          (acc, { moduleName, source }) => ({
+            ...acc,
+            [`${moduleName}.js`]: source,
+          }),
+          {}
+        ),
+      }),
+      babel(babelConfig),
+    ],
+  });
+
+  await Promise.all(
+    BUNDLE_FORMATS.map(({ format, dir }) => {
+      return bundle.write({
+        format,
+        dir,
+      });
+    })
+  );
+}
 
-build({ cwd: path.resolve(__dirname, '../') }).catch(error => {
-  console.log(error);
+build().catch(error => {
+  console.error(error);
```

Comment:
I think we might need:

```suggestion
  // eslint-disable-next-line no-console
  console.error(error);
```

Reply:
aye yi yi -- lemme get those outta there